screens/CenterBtn.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { View, Text } from "react-native";
+import { specificStyles } from "../styles";
+
+//Button to center user on map. Useful if they toggle their location back and forth, as map does not automatically re-center
+const CenterBtn = ({ locationAccess, setMapRegion }) => {
+  const buttonAction =
```

Comment:
I would make the name of this method more specific so it's clear at a glance what it does. Something like "centerMap."